server/src/grpc/server.rs

```diff
@@ -357,58 +450,4 @@ mod tests {
         println!("{:?}", result);
         assert_eq!(result.released, true);
     }
-
```

Comment:
New tests are missing for the newly added functions

Reply:
https://github.com/Arrow-air/svc-compliance/pull/11/commits/87b60b28c50e34eac6a06a06f1ae6476b4e291c1